### DISCUSSION TITLE
Question about Adding Custom Columns in Traces on Langfuse Dashboard

### DISCUSSION THREAD

**Discussion by SurajKrSmitch** [langfuse/langfuse#4681] [Support] [answered]
Hi,
  I’m working on integrating Langfuse with my application and have a question regarding the trace functionality. Specifically, I want to add a custom column to the traces on the Langfuse dashboard.

Is it possible to create and display a custom column in the trace view? I want to dynamically update this column (e.g., user ratings and comments) when a certain API is called in my application.

I've been looking into how to use langfuse_context.update_current_trace for adding metadata, but I am unsure if it allows for the creation of custom columns or if there is a specific way to configure this within the Langfuse dashboard.

If anyone has experience with this or can point me to relevant documentation, I would greatly appreciate it!

**Comment by jannikmaierhoefer** [comment, marked as answer]
Hi @SurajKrSmitch, happy to help! You can create add a score to your trace via `langfuse_context.update_current_trace`. Scores can be either Numeric, Categorical, or Boolean. This score can be updated dynamically. You can then add this score as a column in the trace table. Would this be an option for you?

**Reply by marcklingen** [reply]
Documentation: https://langfuse.com/docs/scores/custom
